packages/contracts/src/contracts/current/protocol/Exchange/MixinMatchOrders.sol

```diff
@@ -224,4 +228,89 @@ contract MixinMatchOrders is
         // Return fill results
         return matchedFillResults;
     }
+
+    /// @dev Settles matched order by transferring appropriate funds between order makers, taker, and fee recipient.
+    /// @param leftOrder First matched order.
+    /// @param rightOrder Second matched order.
+    /// @param takerAddress Address that matched the orders. The taker receives the spread between orders as profit.
+    /// @param matchedFillResults Struct holding amounts to transfer between makers, taker, and fee recipients.
+    function settleMatchedOrders(
+        LibOrder.Order memory leftOrder,
+        LibOrder.Order memory rightOrder,
+        address takerAddress,
+        LibFillResults.MatchedFillResults memory matchedFillResults
+    )
+        internal
```

Comment:
Can be lowered to `private` since only this contract ever calls it (and only this one should call it).

Reply:
We can also inline it in the one place where it is called, but only if it does not hit the variable limit or make the code too long.